### DISCUSSION TITLE
augmented reality does not work (no icons appear)

### DISCUSSION THREAD

**Discussion by Jackblack210** [google/model-viewer#2846] [Q&A]
Hi guys, I have a problem that when passing the Model-viewer component to an element (Elementor-Wordpress) only the 3d viewer works but not the augmented reality on cell phones (IOS and android), I tried it on the glitch page and it didn't work either, now the model is hosted on my server, I detail what I did so that they have an idea (my knowledge is at the level of youtube xD tutorials):

+ I pass the already modified Snippet to the Elementor HTML element.
+ I pass the Model-viewer component as well.
+ I add a style and basic metadata for this case.
--
- I do not see the icon to see in your space (Augmented reality)

Thank you.

![00000000](https://user-images.githubusercontent.com/67923535/136431855-852aabe7-c105-4e91-a2b6-a3ff5bbe65ae.png)

**Comment by ghost** [comment]
Do you have [AR attributes enabled](https://modelviewer.dev/examples/augmentedreality/#webXR) `ar ar-modes="webxr scene-viewer quick-look"`
Other than that, if you can share a glitch page that repros the issue it would be helpful
